presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h

```diff
@@ -46,7 +46,8 @@ struct LocalShuffleInfo {
 /// multi-process use scenarios as long as each producer or consumer is assigned
 /// to a distinct group of partition IDs. Each of them can create an instance of
 /// this class (pointing to the same root path) to read and write shuffle data.
-class LocalPersistentShuffle : public ShuffleInterface {
+class LocalPersistentShuffle : public ShuffleReadInterface,
```

Comment:
For cosco shuffle implementation, let's separate the reader and writer implementations. Thanks!

Reply:
Yeah that is the intent